docs/source/markdown/podman-machine-init.1.md

```diff
@@ -55,6 +55,17 @@ Memory (in MB).
 
 Start the virtual machine immediately after it has been initialized.
 
+#### **--volume**, **-v**=*source:target*
+
+Mounts a volume from source to target.
+
+Create a mount. If you specify /host-dir:/machine-dir, Podman mounts _host-dir_ in the host to _machine-dir_ in the Podman machine.
```

Comment:
As a practice, we avoid pronouns in the man pages:
```suggestion
Create a mount. If /host-dir:/machine-dir is specified as the `*source:target*`, Podman mounts _host-dir_ in the host to _machine-dir_ in the Podman machine.
```

Reply:
Will change these in the other PR